rules/library.bzl

```diff
@@ -1106,15 +1106,23 @@ def apple_library(
     if module_map:
         objc_hdrs.append(module_map)
 
+    swift_interop_hint_name = name + "_swift_interop_hint"
+    swift_interop_hint(
```

Comment:
AIUI using `swift_interop_hint` is preferred over the `objc_library` `module_map` attr, so this seems good -- but is this necessary for rules_swift 3.x?

Reply:
`module_map` is now "deprecated" in `objc_library` so it fails when using rules_swift 3.x: https://github.com/bazelbuild/rules_swift/blob/master/swift/swift_clang_module_aspect.bzl#L220-L225